app/api/api_v1/schemas/search.py

```diff
@@ -68,7 +68,8 @@ class SearchRequestBody(BaseModel):
     limit: int = 10  # TODO: decide on default
     offset: int = 0
 
-    continuation_token: Optional[str] = None
+    continuation_tokens: Optional[Sequence[str]] = None
+    continuation_token: Optional[str] = None  # Deprecated
```

Comment:
Hey hey, mind ELI5ing this for me? How come we need to keep it in if it's deprecated?

Reply:
It's probably not necessary, I wasn't sure if it was being used so thought I'd do all the additive changes here and then do all the breaking changes to the api as a [separate PR](https://github.com/climatepolicyradar/navigator-backend/pull/238)